lightly/api/api_workflow_client.py

```diff
@@ -122,6 +126,24 @@ def dataset_id(self) -> str:
                                       f"taking the last modified dataset {last_modified_dataset.name} as default dataset."))
             return self._dataset_id
 
+    @dataset_id.setter
+    def dataset_id(self, dataset_id: str):
+        """Sets the current dataset id for the client.
+        
+        Args:
+            dataset_id:
+                The new dataset id.
+
+        Raises:
+            ValueError if the dataset id does not exist.
+        """
+        all_datasets: List[DatasetData] = self._datasets_api.get_datasets()
+        for dataset in all_datasets:
+            if dataset.id == dataset_id:
+                self._dataset_id = dataset_id
+                return
+        raise ValueError(f"A dataset with the id {dataset_id} does not exist on the web platform.")
```

Comment:
we should probably go for a consistent name for the "web platform". @MalteEbner used server.

Reply:
I changed it to "Lightly Platform" because we use this everywhere in the docs.